Description of the backup emergency kit (#36543)
home-assistant/home-assistant.github.io: source/more-info/backup-emergency-kit.markdown

---
title: "Backup emergency kit"
description: "Information about backup emergency kit"
---

> The backup emergency kit contains information needed to recover your backup, such as the encryption key.

Reviewer: Not “such as”. It is the encryption key.

Reviewer 2: It was changed very recently whatsoever, https://github.com/home-assistant/frontend/commit/637fe37ef4c46faf34dfa8693943a806c9758de1

> The backup emergency kit contains information needed to recover your backup, such as the encryption key.

Author: ```suggestion
The backup emergency kit contains the encryption key needed to recover your backup.
```

> Backups are encrypted (AES-128) and you need this information to decrypt the backup when you want to restore  your system from that backup.

Reviewer 3: ```suggestion
Backups are encrypted (AES-128) and you need this information to decrypt the backup when you want to restore your system from that backup.
```

## Storing the backup emergency kit somewhere safe

1. Download the emergency kit:
   - directly when setting up a backup
   - or any time from the settings page.

    ![Screenshot showing the encryption key in the download dialog for the backup emergency kit](/images/more-info/backup_emergency_kit_01.png)

2. Store the kit somewhere safe, outside of the Home Assistant system.
   - Without the encryption key, there is no way to restore the backup.

> ## Changing the encryption key

Author: ```suggestion
## Generating a new encryption key
```

> The first time when you set up your backups an encryption key is be generated. But you can generate new ones. Old backups will still have the generated encryption key, but new backups will use the encryption key you defined yourself.

Author: ```suggestion
The first time when you set up your backups, an encryption key is generated. But you can generate new ones. Old backups will still have the generated encryption key, but new backups will use the encryption key you defined yourself.
```

> The first time when you set up your backups an encryption key is be generated. But you can create your own key. Old backups will still have the generated encryption key, but new backups will use the encryption key you defined yourself.

Reviewer 3: This wording is still odd, as you can't define your own.

```suggestion
The first time when you set up your backups an encryption key is be generated. But you can generate new ones. Old backups will still have the generated encryption key, but new backups will use the encryption key you defined yourself.
```

---
title: "Backup emergency kit"
description: "Information about backup emergency kit"
---

The backup emergency kit contains the encryption key needed to recover your backup.
Backups are encrypted (AES-128) and you need this information to decrypt the backup when you want to restore your system from that backup.

## Storing the backup emergency kit somewhere safe

1. To download the emergency kit, go to {% my backup title="**Settings** > **System** > **Backups**" %}
2. If it is your first time defining backup settings, select **Setup automatic backup** and download the encryption key.
   - You can also download the key again later under configuration.

    ![Screenshot showing the encryption key in the download dialog for the backup emergency kit](/images/more-info/backup_emergency_kit_01.png)

3. Store the kit somewhere safe, outside of the Home Assistant system.
   - Without the encryption key, there is no way to restore the backup.

## Generating a new encryption key

> The first time when you set up your backups, an encryption key is generated. But you can generate new ones. Old backups will still have the generated encryption key, but new backups will use the encryption key you defined yourself.

Reviewer 2: Reading this, "generating" may not be the correct term to summarize this paragraph. The UI speaks about changing and to me it's more about replacing your encryption key. 

The paragraph can also be rephrased differently:
* since you cannot generate (multiple) new ones, but there is only one encryption key in place at a certain time. 
* old refers here to the currently already existing backup's, while new refers to backups that will be created after you replaced your encryption key. 
* there's no option to define something yourself, the encryption key is generated and presented to you.

Author: right. Thank you for the catch! Rephrased the section.
About generating: I see the issue, but I'm not sure about "replacing" because the old key still applies to the old backups.

1. To generate a new encryption key, go to {% my backup title="**Settings** > **System** > **Backups**" %}.
2. Select **Configure automatic backups** and under **Encryption key**, select **Change**.
3. If you haven't downloaded the current encryption key yet, do it now.
4. Select **Change encryption key**.
5. Download the new encryption key and store it in a safe place.

## Checking and disabling backup location

> 1. To generate a new encryption key, go to {% my backup title="**Settings** > **System** > **Backups**" %}.

Reviewer 2: Copy paste error
